src/WorkspaceManager.ts

```diff
@@ -87,10 +87,6 @@ export class WorkspaceManager implements vscode.Disposable {
         }
 
         const resolvedTask = await resolveVariablesAsync(found, varToValue);
-        // Task.name setter needs to be triggered in order for the task to clear its __id field
```

Comment:
`vscode` namespace has a `version` field. would you update your PR that it checks it and makes a decision based on that?
If you say it works with the current version then let's just add and if > 1.72.0.
If someone complains we can adjust :)

Reply:
That sounds good, I just pushed up that change. 😀